### ISSUE TITLE
getting max_connections issue on standy cluster

### ISSUE THREAD

**Issue by nawarajshahi** [CrunchyData/postgres-operator#2952]
We're trying to clone a database using the documentation found [here](https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/disaster-recovery/). 

However, we have to increase the default max_connections on our databases to support our workloads. We did this by using the following spec: 
```
patroni:
    dynamicConfiguration:
      postgresql:
        parameters:
          max_connections: 300
```
However, this seems to cause the cloning process to fail. We're receiving the following error: 
```
FATAL:  hot standby is not possible because max_connections = 100 is a lower setting than on the master server (its value was 300)
```

We also tried with the following configuration, but same error persists. 
```
  patroni:
      dynamicConfiguration:
        postgresql:
          parameters:
            max_connections: 300
        standby_cluster:
          parameters:
            max_connections: 300
        bootstrap:
          dcs:
            postgresql:
              parameters:
                max_connections: 300
```
Any assistance would be greatly appreciated, thank you!

**Comment by nawarajshahi** [comment]
resolved this issue by upgrading the PGO from current version 5.0.2 to 5.0.4

**Comment by cbandy** [comment]
This was likely fixed by #2657 in [v5.0.3](https://access.crunchydata.com/documentation/postgres-operator/v5/releases/5.0.3/).
